compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion.hs

```diff
@@ -687,11 +658,7 @@ convertExpr env0 e = do
     go env (VarIs "fromRational") (LType (isNumLitTy -> Just n) : LExpr (VarIs ":%" `App` tyInteger `App` Lit (LitNumber _ top _) `App` Lit (LitNumber _ bot _)) : args)
         = fmap (, args) $ convertRationalNumericMono env n top bot
     go env (VarIs "fromRational") (LType scaleTyCoRep : LExpr (VarIs ":%" `App` tyInteger `App` Lit (LitNumber _ top _) `App` Lit (LitNumber _ bot _)) : args)
-        = do
-            scaleType <- convertType env scaleTyCoRep
-            fmap (, args) $ convertRationalNumericPoly env scaleType top bot
-
-
+        = unsupported "Polymorphic numeric literal. Specify a fixed scale by giving the type, e.g. (1.2345 : Numeric 10)" ()
```

Comment:
Only catching this during LF conversion seems slightly ugly but admittedly I don’t know how to catch it earlier :disappointed: